### PR TITLE
Some refactor

### PR DESCRIPTION
**Pull request by NejcZdovc**
- [x] Unification of folder structure
- [x] Unification of header based on key map

### CODE REVIEW

File: src/components/keymap/header/keymap-header.component.scss [hunk 1]
```diff
@@ -7,7 +7,7 @@
             cursor: pointer;
 
             &:hover {
-                color: #b9b6b6;
+                color: $icon-hover;
```
Comment (fjozsef) [inline src/components/keymap/header/keymap-header.component.scss]: Editing this file should not be the part of macro refactoring.

Comment (NejcZdovc) [inline src/components/keymap/header/keymap-header.component.scss]: I know, but I don't think that this is something that should have a separate PR.

Comment (fjozsef) [inline src/components/keymap/header/keymap-header.component.scss]: Then rename the PR's title.

Comment (NejcZdovc) [inline src/components/keymap/header/keymap-header.component.scss]: I don't know to what

**Pull request merged**